### PR TITLE
New package: ufraw-0.22

### LINKED ISSUE #7099: Package Request: ufraw
Ufraw is a library to convert digital camera raw image files (like CR2) to more common formats (png, jpeg). 

I tried building it myself but it fails for now, I believe some issue with glib. 

I'm traveling right now so can't share the template. If someone wants to join right in he's free to :)

### PR DESCRIPTION
**Pull request by Johnnynator**
closes #7099

### CODE REVIEW

**Comment by lemmi** [comment]
I was just checking this out, but SourceForge doesn't seem to let me download the distfile atm. I just get a bunch of redirects. I will look at it the next couple of days again.

File: srcpkgs/ufraw/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'ufraw'
+pkgname=ufraw
+version=0.22
+revision=1
+build_style=gnu-configure
+configure_args="--enable-contrast --enable-mime --enable-extras --enable-openmp --with-gimp"
+hostmakedepends="pkg-config"
+makedepends="exiv2-devel gimp-devel glib-devel gtkimageview-devel lcms2-devel lensfun-devel"
```
Comment (lemmi) [inline srcpkgs/ufraw/template]: I'm not sure about the value, but you could add jpeg2000 support via libjasper. If you have gimp installed, that lib is on your system anyway.

**Comment by Johnnynator** [comment]
@lemmi I just set extras as optional build_option since probably no one will use nikon-curve and also it is recommended in the README for packaging to not enable extras.
Builds fail because of SourceForge.

**Comment by lemmi** [comment]
+1 for using build_options.

**Comment by cr6git** [comment]
Good job !

**Pull request closed without merging**